### PR TITLE
quassel: don't bundle breeze icons, offer ldap build option

### PR DESCRIPTION
**Pull request by Piraty**
#4854

### CODE REVIEW

File: srcpkgs/quassel/template [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+ -DUSE_QT5=ON -DWITH_BUNDLED_ICONS=OFF"
 hostmakedepends="pkg-config qt5-host-tools qt5-qmake"
 makedepends="qt5-tools-devel phonon-qt5-devel qca-qt5-devel zlib-devel
  libdbusmenu-qt5-devel qt5-script-devel qt5-plugin-tds
- qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-mysql qt5-plugin-odbc"
+ qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-mysql qt5-plugin-odbc
+ qt5-multimedia-devel $(vopt_if ldap libldap-devel)"
 depends="qt5-plugin-sqlite quassel-client-shared>=${version}_${revision}"
 _desc="Modern, cross-platform, distributed graphical IRC client"
 short_desc="${_desc} - standalone client"
```
Comment (maxice8) [inline srcpkgs/quassel/template]: Mind fixing license please ?

Comment (Piraty) [reply srcpkgs/quassel/template]: fixed to `GPL-2.0-only or GPL-3.0-only`

Comment (maxice8) [reply srcpkgs/quassel/template]: don't think we have the `or` keyword defined in our Manual

File: srcpkgs/quassel/template [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
+license="GPL-2.0-only or GPL-3.0-only"
 homepage="https://www.quassel-irc.org"
 distfiles="https://quassel-irc.org/pub/quassel-${version}.tar.bz2"
 checksum=d9822002de5bb1fd1002cccd537e5a0bc8ec365d355fdff4fb7818daefade976
 
+build_options="ldap"
+
 post_install() {
 	vsv quasselcore
 }
```
Comment (maxice8) [inline srcpkgs/quassel/template]: quassel-client-shared should be noarch

**Pull request merged**